packages/api/src/platforms/vtex/resolvers/query.ts

```diff
@@ -273,4 +274,22 @@ export const Query = {
       address,
     }
   },
+  redirect: async (
+    _: unknown,
+    { term, selectedFacets }: QueryRedirectArgs,
+    ctx: Context
+  ) => {
+    if (!term && (!selectedFacets || !selectedFacets.length)) {
```

Comment:
Is this always the case that when there's no term or facets, there's no redirect? Could you add a comment to any docs containing that info?

Reply:
Icaro! Thank you for your input.

We where taking a look at schema.org with @beatrizmaselli and only could ever find close to this was https://schema.org/SearchAction maybe we could do `location` for term and `object` for selectedFacets?
If not, I think that renaming the query itself to `searchRedirect` could make the whole implementation more self-explanatory.

Regarding your question about line 282. Yes, term is just the query param of searches; and selectedFacets is a recollection of dynamic catalog specifications plus departments and subcategories. Also, those are the only 2 conditions available in the admin panel for search redirects